### PR TITLE
Improve the `Node.add_child_below_node()` documentation

### LINKED ISSUE #2730: add_child and add_child_below_node dont use the same syntax and it's quite confusing
https://docs.godotengine.org/en/3.1/classes/class_node.html#class-node-method-add-child

>     void add_child ( Node node, bool legible_unique_name=false )
> 
> Adds a child node. Nodes can have any number of children, but every child must have a unique name. Child nodes are automatically deleted when the parent node is deleted, so an entire scene can be removed by deleting its topmost node.
> 
> Setting legible_unique_name to true creates child nodes with human-readable names, based on the name of the node being instanced instead of its type.
> 
>     void add_child_below_node ( Node node, Node child_node, bool legible_unique_name=false )
> 
> Adds a child node. The child is placed below the given node in the list of children.
> 
> Setting legible_unique_name to true creates child nodes with human-readable names, based on the name of the node being instanced instead of its type.

Here: "add_child ( Node node," <- the node that gets added is the first argument and called "node",
here: "add_child_below_node ( Node node, Node child_node, <- the first argument is also called "node" like above, but the node that gets added is not this one but the second argument called "child_node"

Both descriptions in the first line say "Adds a child node."

If only one of those method descriptions would exist, they would be clear, but together they are confusing because of the changing syntax. It's not clear what argument is for the node I want to add, which one is for the node that gets something appended to.

**Proposal:**
void add_child_below_node ( Node **appending node**, Node **node**, bool legible_unique_name=false )

 **Adds a child node below the appending node.**
Setting legible_unique_name to true creates child nodes with human-readable names, based on the name of the node being instanced instead of its type.

### PR DESCRIPTION
**Pull request by Calinou**
This closes https://github.com/godotengine/godot-docs/issues/2730.

cc @golddotasksquestions

### CODE REVIEW

**Comment by golddotasksquestions** [comment]
Thank you!

File: doc/classes/Node.xml [hunk 1]
```diff
@@ -129,21 +129,23 @@
...
 				[/codeblock]
+				If you need the child node to be added below a specific node in the list of children, use [method add_child_below_node] instead of this method.
 				[b]Note:[/b] If you want a child to be persisted to a [PackedScene], you must set [member owner] in addition to calling [method add_child]. This is typically relevant for [url=https://godot.readthedocs.io/en/latest/tutorials/misc/running_code_in_the_editor.html]tool scripts[/url] and [url=https://godot.readthedocs.io/en/latest/tutorials/plugins/editor/index.html]editor plugins[/url]. If [method add_child] is called without setting [member owner], the newly added [Node] will not be visible in the scene tree, though it will be visible in the 2D/3D view.
 			</description>
 		</method>
 		<method name="add_child_below_node">
 			<return type="void">
 			</return>
-			<argument index="0" name="node" type="Node">
+			<argument index="0" name="appending_node" type="Node">
```
Comment (akien-mga) [inline doc/classes/Node.xml]: I don't know if `appending_node` is clear enough, it doesn't sound immediately obvious to me. Maybe "sibling" would be better? Or "preceding_node".

Comment (asheraryam) [reply doc/classes/Node.xml]: I was thinking the same thing, `appending_node` is almost as opaque as no name at all. `preceding_node` is immediately clear. `sibling` also isn't indicative of the purpose.

Comment (golddotasksquestions) [reply doc/classes/Node.xml]: Let's change it to `preceding_node ` then.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**